example/App.js

```diff
@@ -71,15 +68,16 @@ export default class ProgressChart extends React.Component {
           backgroundWidth={5}
           fill={fill}
           tintColor="#00e0ff"
-          onAnimationComplete={() => console.log('onAnimationComplete')}
```

Comment:
I'd like to keep this as part of example since it is useful to see that the feature actually works while running the app

Reply:
Cool with me 👍 Any other thoughts?